bundler.d/development.rb

```diff
@@ -15,4 +15,5 @@
   gem 'spring', '>= 1.0', '< 3'
   gem 'benchmark-ips'
   gem 'foreman'
+  RUBY_VERSION >= '2.3' && gem('bootsnap', :require => false)
```

Comment:
Please use if, we are not in bash here :)

Reply:
But, but ... but hounds says yay! :)

Reply:
And still 👎 from this 🐶